src/orbital-demo/src/app/core/services/services/pets.service.ts

```diff
@@ -0,0 +1,217 @@
+/* tslint:disable */
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpRequest, HttpResponse, HttpHeaders } from '@angular/common/http';
+import { BaseService as __BaseService } from '../base-service';
+import { ApiConfiguration as __Configuration } from '../api-configuration';
+import { StrictHttpResponse as __StrictHttpResponse } from '../strict-http-response';
+import { Observable as __Observable } from 'rxjs';
+import { map as __map, filter as __filter } from 'rxjs/operators';
+
+import { Pets } from '../models/pets';
+@Injectable({
+  providedIn: 'root',
+})
+class PetsService extends __BaseService {
+  static readonly listPetsPath = '/pets';
```

Comment:
Looks like this set of variables isn't even used 🤔

Reply:
Also... seems like this whole file I auto-generated, so might not be worth cleaning it up?

Reply:
It was all auto-generated - if we want to clean it up that's cool, but since this is a demo app we don't really need to refactor boilerplate unless it limits readability or performance.